formatter/formatter_test.go

```diff
@@ -157,10 +223,8 @@ test.go
 			if len(stdout) > 0 {
 				t.Errorf("formatter wrote to stdout: %q", stdout)
 			}
-			got := strings.TrimSpace(output)
-			want := strings.TrimSpace(td.want)
-			if got != want {
-				t.Errorf("got %q, want %q", got, want)
+			if diff := cmp.Diff(td.want, output); diff != "" {
```

Comment:
Does this PR and test file worth adding google/cmp to revive dependency?

I feel like it's overkill for doing a diff on a string.

Yes, the result will be better if something differs.

But adding a dependency for something that, sounds overkill and unnecessary.

> [!NOTE]
> I night have misunderstood why you ads it, or maybe you want to reuse it everywhere